packages/teams_memory/teams_memory/services/llm_service.py

```diff
@@ -66,27 +66,30 @@ def __init__(self, config: LLMConfig):
     @overload
     async def completion(
         self,
-        messages: List,
+        messages: List[Dict[str, str]],
         response_model: None = None,
         override_model: Optional[str] = None,
-        **kwargs,
+        stream: bool = False,
```

Comment:
do we need this?

Reply:
Nope. mypy's out to make me look crazy. Removing.